test/e2e/test/version.go

```diff
@@ -16,9 +16,9 @@ const (
 	// LatestReleasedVersion6x is the latest released version for 6.x
 	LatestReleasedVersion6x = "6.8.23"
 	// LatestReleasedVersion7x is the latest released version for 7.x
-	LatestReleasedVersion7x = "7.17.4"
+	LatestReleasedVersion7x = "7.17.5"
```

Comment:
There is also a `7.17.x` to update here: https://github.com/elastic/cloud-on-k8s/blob/d23142646ff9e7af1bcb4d6a1a7150ee245ae398/.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile#L104